spec/factories/user.rb

```diff
@@ -0,0 +1,15 @@
+FactoryGirl.define do
+  factory :user do
+    name 'Test user'
+    role 'user'
+    email 'example@mail.com'
+    password 'passw_1234'
+
+    factory :user_admin do
```

Comment:
there is no point in this nested factory since you override every single field. either create 2 independent factories with different values or leave it as it is and override only those fields that really distinguish `admin_user` from `user`